src/main/scala/edu/ie3/simona/model/grid/GridModel.scala

```diff
@@ -496,26 +496,21 @@ case object GridModel {
 
     // build
     // / nodes
-    val nodes: Set[NodeModel] =
-      subGridContainer.getRawGrid.getNodes.asScala
-        .collect { case nodeInput: NodeInput =>
-          NodeModel(nodeInput, startDate, endDate)
-        }
-        .to(collection.immutable.Set)
+    val nodes = subGridContainer.getRawGrid.getNodes.asScala.toSeq.map {
+      nodeInput => NodeModel(nodeInput, startDate, endDate)
+    }
```

Comment:
I think it might make sense to shortly comment on why this is transformed to a sequence as opposed to a set like for all the other grid elements